### PR TITLE
5 get/api/reviews

### PR DESCRIPTION
**Pull request by t-cbailey**
5 get/api/reviews

### CODE REVIEW

File: playground.sql [hunk 1]
```diff
@@ -1,3 +1,16 @@
-\c nc_games_test;
-SELECT * FROM reviews
-WHERE review_id = 1;
\ No newline at end of file
+\c nc_games_test
```
Comment (HannahBurks) [inline playground.sql:1]: Add this to your git ignore :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -38,50 +49,62 @@ describe("App", () => {
...
+            expect(typeof review.owner).toBe("string");
+            expect(typeof review.title).toBe("string");
+            expect(typeof review.review_id).toBe("number");
+            expect(typeof review.category).toBe("string");
+            expect(typeof review.review_img_url).toBe("string");
+            expect(typeof review.created_at).toBe("string");
+            expect(typeof review.votes).toBe("number");
+            expect(typeof review.designer).toBe("string");
+            //I CANNOT make comment count into a number. it just will not!
+            expect(typeof review.comment_count).toBe("string");
```
Comment (HannahBurks) [inline __tests__/app.test.js:68]: That is fine, SQL turns it to a string, however, you can check by adding + on the test side to check once converted it is infact a number or you can just leave it as is :)

Comment (HannahBurks) [reply __tests__/app.test.js:68]: I would add a test to check the comment count is correct though for one of the article ids.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -38,50 +49,62 @@ describe("App", () => {
     });
   });
 
-  describe("/api/reviews/:review_id", () => {
-    test("GET 200- should respond with a review object, of the correct id ", () => {
+  describe("/api/reviews", () => {
```
Comment (HannahBurks) [inline __tests__/app.test.js:52]: You will also need to add a test to ensure they are all sorted - take a look at jest sorted

**Review by HannahBurks — CHANGES_REQUESTED** [review]
Nice start Tim, a few more changes to make to your tests (we need to make sure they are sorted). Once done please send another pull request

**Review by mezzDavies — APPROVED** [review]
Changes approved TIm 👍

**Pull request merged**